CnpSdkForNet/CnpSdkForNet/CnpOnline.cs

```diff
@@ -71,9 +99,9 @@ public CnpOnline(Dictionary<string, string> config)
             remove { _communication.HttpAction -= value; }
         }
 
-        public void SetCommunication(Communications communication)
+        public static void InitializeCommunication(Dictionary<string, string> config)
         {
-            this._communication = communication;
+            _communication = new Communications(config);
         }
```

Comment:
As mentioned above, this methodology bugs me. Any existing CnpOnline will forcefully receive a new _communication (as they all reference the same one), and thus will get a new config.

Perhaps Communications can be a utility class, and keep configs in the parameters? CnpOnline can then hold onto a config itself, and would not need to hold a separate merchant id. Keep in mind I do not know the architecture of this SDK well enough so this may have unintended side effects.